src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedController.java

```diff
@@ -101,6 +115,16 @@
                         .build());
     }
 
+    private void cleanupCaseDataBeforeSubmittingToCcd(CaseDetails caseDetails) {
+        // Must remove any added case data as CCD will return an error
+        caseDetails.getData().remove("caseNumber");
+        caseDetails.getData().remove("reference");
+        caseDetails.getData().remove("addressee");
```

Comment:
When was this data added?

Reply:
This data has to be added so Docmosis can generate the documents correctly, but if the data is left in it causes CCD to fall over. Therefore we must cleanup the casedata after generating the documents